examples/rtl-tests.js

```diff
@@ -288,19 +288,46 @@ const LARGE_TESTS = [
 ];
 var docDefinition = {
 	content: [
-		generateTestHeader("Test nesting"),
-		generateTestString(
-			"Comment from an alert, lifted directly from digest-pdf-export"
-		),
-		commentFromDigestPdfExport,
-		newline(),
-		generateTestHeader("Arabic vs LTR fonts"),
-		...ARABIC_TESTS,
-		generateTestHeader("Hebrew vs LTR fonts"),
-		...HEBREW_TESTS,
-		generateTestHeader("All scripts megamix"),
-		...LARGE_TESTS,
+		// generateTestHeader("Test nesting"),
```

Comment:
Did you mean to leave these commented out?

Reply:
The tests are taking a hammering. I didn't mean to.